plugin/pkg/scheduler/algorithm/predicates/metadata_test.go

```diff
@@ -338,21 +339,18 @@ func TestPredicateMetadata_AddRemovePod(t *testing.T) {
 		existingPodsMeta1, nodeInfoMap := getMeta(schedulertesting.FakePodLister(test.existingPods))
 		// Add test.addedPod to existingPodsMeta1 and make sure meta is equal to allPodsMeta
 		nodeInfo := nodeInfoMap[test.addedPod.Spec.NodeName]
-		if err := existingPodsMeta1.AddPod(test.addedPod, nodeInfo); err != nil {
-			t.Errorf("test [%v]: error adding pod to meta: %v", test.description, err)
-		}
-		if err := predicateMetadataEquivalent(allPodsMeta, existingPodsMeta1); err != nil {
-			t.Errorf("test [%v]: meta data are not equivalent: %v", test.description, err)
-		}
+		err := existingPodsMeta1.AddPod(test.addedPod, nodeInfo)
+		assert.NoError(t, err, "test [%v]: error adding pod to meta: %v", test.description, err)
```

Comment:
FWIW: I don't think we should be using `assert`. We should be using `require` instead, here and elsewhere. ```The require package provides same global functions as the assert package, but instead of returning a boolean result they terminate current test.```

Reply:
I can make those changes